handler_templates.ts

```diff
@@ -380,8 +380,8 @@ function unmapKeys(obj: {[key: string]: any}, schema?: Schema): object {
 
 export function untransformBody(body: any, schema: Schema | undefined): any {
```

Comment:
Looks like we had a bug here compared to what `transformBody` does (cc @chris-codaio).

Reply:
How does this manifest itself? Will this break any existing behavior related to 2-way sync?

Reply:
It looks safe to me given how we're calling it in the main repo. The only case I have concerns about is here:

https://github.com/coda/coda/blob/8b056761fdb68f55c89e54c5688a2e066f62f302/modules/server/calc-server/private/calc_service.ts#L2408-L2415

I wasn't able to get this codepath to trigger locally though - tried refreshing both pack sync tables and cross-doc.

Reply:
Wasn't able to repro that code path with 2-way sync but I did update one place where we have test coverage here: https://github.com/coda/coda/pull/118340